x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/Security.java

```diff
@@ -409,6 +409,11 @@
         "oidc",
         License.OperationMode.PLATINUM
     );
+    public static final LicensedFeature.Persistent JWT_REALM_FEATURE = LicensedFeature.persistent(
+        REALMS_FEATURE_FAMILY,
+        "jwt",
+        License.OperationMode.PLATINUM
+    );
```

Comment:
I guess you haven't update the PR yet because this file needs the extra Rest Header change.

Reply:
Resolved. Yes, when we found the issue it close to midnight for me. I added the request header to the accept list first thing next morning in commit 0a50dc9e5d534ed1a4a02cce1d9e68e80472a20c.